src/utils/entitySpec.js

```diff
@@ -15,6 +15,26 @@ const toLowerCaseMap = {
   mavencentralsource: NONE
 }
 
+const NPM_WEBSITE = 'npmjs.com'
```

Comment:
Should also add npmjs.org (it maps to .com). This is another reason to have this as an object. We could then just plug the same object in twice and have it handle both cases.